internal/nodes/avi_model_advl4_translator.go

```diff
@@ -551,9 +551,17 @@ func (o *AviObjectGraph) ConstructSharedVipSvcLBNode(sharedVipKey, namespace, ke
 		vsVipNode.BGPPeerLabels = lib.GetGlobalBgpPeerLabels()
 	}
 
+	t1lr := lib.GetT1LRPath()
+	if t1lr != "" {
```

Comment:
If it's empty and T1lr is defined in the AviInfraSetting, vrfcontext will not be set to empty string. We should also check if t1lr is defined in the aviinfrasetting cr and based on that set the vrfcontext and t1lr fields accordingly.

Reply:
oh, I see you have updated the buildWithInfraSetting function, please ignore the above comment